SU-HSDO/suhumsci
# shs-87: set up base reduced motion option

## Summary
This adds `prefers-reduced-motion: reduce` to all elements.

## Steps to Test
1. Go to a Colorful page with animations. Here is a good example: `components/hero-images-main-region`. Verify animations play (transitions, sliders, etc.)
2. Depending on what OS you use, turn on Reduced Motion in your Accessibility preferences. For Macs, open Accessibility -> Display. A screenshot is included below.
3. Visit the same page from above. Verify animations/transitions do not play but everything is still functional.
4. Do The same for Traditional.
5. Poke around a few other pages and verify functionality still works. I tried about 20 pages, and everything seems fine. While this touches a lot of things, it's low risk due to the < 0.1% of people who would have this setting on. 

## Screenshots
![Screen Shot 2022-11-29 at 3 40 55 PM](https://user-images.githubusercontent.com/8405274/204643593-477576af-7f3d-44c0-a7c8-97d07ed72d8e.png)

## Notes
- Will need to change base branch after sprint branch is set up
- The ticket directly mentioned doing this on the slider, but it makes more sense to apply to *all* transitions and animations.
